src/docs/development/tools/sdk/upgrading.md

```diff
@@ -10,7 +10,7 @@ Flutter SDK or the packages that your app depends on.
 
 ## Upgrading the Flutter SDK
 
-To update both the Flutter SDK use the `flutter upgrade` command:
+To update both the Flutter SDK and the packages that your app depends on, use the `flutter upgrade` command:
```

Comment:
"flutter upgrade" upgrades the Flutter SDK, but it doesn't upgrade your pub dependencies. I am not quite sure what you mean by this change.

Reply:
Ok @sfshaza2! I guess, it's the "both" that should be removed then. The phrase, should be "To update the Flutter SDK use..." (with out the "both"). "Both" what? Get me?